zcash_client_sqlite/src/testing/pool.rs

```diff
@@ -1532,12 +1526,6 @@ pub(crate) fn checkpoint_gaps<T: ShieldedPoolTester>() {
     // Scan the block
     st.scan_cached_blocks(account.birthday().height() + 10, 1);
 
-    // Fake that everything has been scanned
```

Comment:
I'm a little bit concerned that this test still passes without this.

Reply:
I verified that without the `scan_cached_blocks` it fails, so it needs that specific block to be scanned, but apparently not the rest of the chain.